test/nodes/test_reader.py

```diff
@@ -250,6 +254,58 @@ def test_farm_reader_update_params(docs):
         reader.predict(query="Who lives in Berlin?", documents=docs, top_k=3)
 
 
+@pytest.mark.integration
+def test_farm_reader_load_diff_ways(docs):
+    # There are x different ways to load a FARMReader model.
+    # 1. HuggingFace Hub (online load)
+    # 2. HuggingFace downloaded (local load)
+    # 3. HF Model saved as FARM Model (same works for trained FARM model) (local load)
+    # 4. FARM Model converted to transformers (local load)
+    # 5. ONNX Model load (covered by test_farm_reader_onnx_conversion_and_inference)
+
+    # Test Case: 1. HuggingFace Hub (online load)
+    reader = FARMReader(
+        model_name_or_path="deepset/minilm-uncased-squad2", use_gpu=False, no_ans_boost=0, num_processes=0
+    )
+
+    # original reader
+    prediction = reader.predict(query="Who lives in Berlin?", documents=docs, top_k=3)
+    assert len(prediction["answers"]) == 3
+    assert prediction["answers"][0].answer == "Carla"
```

Comment:
I don't think we should test the reader's predictions here (this is already tested in `test_output`).